File: 2023/day3/main-part1.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
    std::ifstream file("test_input.txt");

    std::string line;
    while (std::getline(file, line)) {
        std::cout << line << std::endl;
    }

    file.close();
}

File: 2023/day12/main-part1.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
    file, _ := os.Open("test_input.txt")
    defer file.Close()

    totalPermutations := 0

    scn := bufio.NewScanner(file)

    for scn.Scan() {
        line := scn.Text()
        // permutation := 0

        fmt.Printf("%q\n", strings.Split(line, " "))
    }

    fmt.Println("Final ans: ", totalPermutations)
}
